.azure/templates/spinxsk.yml

```diff
@@ -27,10 +27,14 @@ parameters:
 - name: xdpmpPollProvider
   default: 'NDIS'
 - name: osName
+- name: enableEbpf
+  type: boolean
+  default: false
 
 jobs:
-- job: spinxsk_${{ parameters.arch }}_${{ parameters.config }}_${{ parameters.osName }}_${{ replace(parameters.pool, '-', '_') }}
-  displayName: spinxsk-${{ parameters.osName }}-${{ parameters.arch }}_${{ parameters.config }} (${{ parameters.pool }})
+- job: spinxsk_${{ parameters.arch }}_${{ parameters.config }}_${{ parameters.osName }}_${{ replace(parameters.pool, '-', '_') }}_${{ replace(replace(parameters.enableEbpf, 'True', 'Ebpf'), 'False', '') }}
```

Comment:
nit, but I think this is slightly better:
```suggestion
- job: spinxsk_${{ parameters.arch }}_${{ parameters.config }}_${{ parameters.osName }}_${{ replace(parameters.pool, '-', '_') }}${{ replace(replace(parameters.enableEbpf, 'True', '_Ebpf'), 'False', '') }}
```

Reply:
I indirectly addressed this during a refactor, I believe.